SwiftyAttributes/Sources/common/Attribute.swift

```diff
@@ -12,15 +12,24 @@
     public typealias Font = NSFont
     public typealias Cursor = NSCursor
     public typealias TextAlternatives = NSTextAlternatives
+    public typealias Point = NSPoint
+    public typealias Size = NSSize
+    public typealias Rect = NSRect
+    public typealias DrawingOptions = NSString.DrawingOptions
 #else
     import UIKit
     public typealias Color = UIColor
     public typealias Font = UIFont
+    public typealias Point = CGPoint
+    public typealias Size = CGSize
+    public typealias Rect = CGRect
+    public typealias DrawingOptions = NSStringDrawingOptions
```

Comment:
could we maybe move these to the files where they're used? The other `typealiase`s are declared here because they're used in this file